_posts/2023-12-24-update.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Week Ending December 24, 2023
+date: 2023-01-26 22:00:00 -0000
+slug: 2023-01-24-update
+---
+
+## Developer News
+
+SIG Release [updated](https://groups.google.com/a/kubernetes.io/g/dev/c/JfZQhip-R7U) it's roadmap for 2024 onwards. If you want to contribute, just reach out to SIG Release using the K8s Slack (#sig-release) or comment on any of the open GitHub issues.
+
+## Release Schedule
+
+**Next Deadline: 1.30 Cycle Begins, January 2024**
+
+We are in the holiday period between releases; dates for 1.30 have not been published yet. If you want to become a part of the [Release team](https://github.com/kubernetes/sig-release/issues/2377), apply soon!
+
+## Featured PRs
+
+
+## KEP of the Week
+
+
+## Other Merges
+
+*
+
+## Promotions
+
+*
+
+## Deprecated
+
+*
+
+## Version Updates
+
+*
+
+## Subprojects and Dependency Updates
+
```

Comment:
```suggestion

*  [cri-io/cri-io to 1.29.0](https://github.com/cri-o/cri-o/releases/tag/v1.29.0)  is released 
```

Reply:
@mfahlandt I was working on this right now! I will push the changes and ping you for a review